src/makeCalculator.js

```diff
@@ -37,7 +37,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
```

Comment:
don't be shy with variable names
```suggestion
  const calculator = {
```